--- a/kfp_authentication.py	2023-06-09 10:13:11.412910808 -0400
+++ b/kfp_authentication.py	2023-06-09 10:14:39.879565175 -0400
@@ -230,6 +230,7 @@
         """
 
         kf_url = urlsplit(api_endpoint)._replace(path="").geturl()
+        kf_pipelines_ssl_sa_cert = os.getenv("PIPELINES_SSL_SA_CERTS", None)
 
         # return data structure for successful requests
         auth_info = {
@@ -239,6 +240,7 @@
             "cookies": None,  # passed to KFP SDK client as "cookies" param value
             "credentials": None,  # passed to KFP SDK client as "credentials" param value
             "existing_token": None,  # passed to KFP SDK client as "existing_token" param value
+            "ssl_ca_cert": kf_pipelines_ssl_sa_cert,  # passed to KFP SDK Client as "ssl_ca_cert" param value
         }
 
         try:
